### PR TITLE
Local development tweaks

### PR DESCRIPTION
**Pull request by ce-rpf**
Just a couple of small tweaks to make local development easier:

* `disableHostCheck` facilitates using the `rpfdev.com` domain I'm working on (I'll share more details of this once its ready for use!)
* adding shared network facilitates a local nginx proxy
* changing port avoids a clash with projects-ui

### CODE REVIEW

**Comment by loiswells97** [comment]
Can we have the main editor site on `3010` too? I think might be clashing with something else as well?

**Comment by ce-rpf** [comment]
Main editor now on 3010 :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Moved web component custom events from the `editor-wc` element to the `document` (#710)
 - Renamed web component custom events to be prefixed with `editor-` (#710)
 - Switch props of `WebComponentLoader` from `snake_case` to `camelCase` (#712)
+- Move local development port to avoid projects-ui clash (#736)
```
Comment (sra405) [inline CHANGELOG.md:34]: ⛏️ can we move this to the `Unreleased` section?

Comment (ce-rpf) [reply CHANGELOG.md]: 🤦 done, sorry!

**Review by sra405 — APPROVED** [review]
👍 finally no more port clashes!!

**Pull request merged**